### PR TITLE
UIDEXP-337: Display duplicate SRS records in the Failed column

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
[UIDEXP-337](https://issues.folio.org/browse/UIDEXP-337)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 1]
```diff
@@ -149,9 +149,28 @@ export const JobLogsContainer = props => {
...
+              }, {
+                failedOther,
+                failedSrs,
+              });
+            case failedSrs > 0 && failedOther === 0:
+              return intl.formatMessage({
+                id: 'ui-data-export.column.errors.duplicates',
+              }, {
+                failedSrs,
+              });
```
Comment (artem-blazhko) [inline src/components/JobLogsContainer/JobLogsContainer.js:170]: Why do you need "failedSrs" param here if translation does not contain it?
Also, I do not see any reason to have a translation for "column.errors.duplicates" if it is an empty string.

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 2]
```diff
@@ -149,9 +149,28 @@ export const JobLogsContainer = props => {
...
         errors: record => {
-          const failed = record.progress?.failed;
-
-          return failed || '';
+          const failedSrs = record.progress?.failed?.duplicatedSrs;
+          const failedOther = record.progress?.failed?.otherFailed;
+
+          switch (true) {
+            case failedSrs === 0 && failedOther > 0:
+              return `${failedOther}`;
```
Comment (artem-blazhko) [inline src/components/JobLogsContainer/JobLogsContainer.js:157]:
```suggestion
              return failedOther;
```

**Review by artem-blazhko — CHANGES_REQUESTED** [review]

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/JobLogsContainer/JobLogsContainer.js]: fixed

File: src/components/JobLogsContainer/JobLogsContainer.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/JobLogsContainer/JobLogsContainer.js:170]: also fixed now

**Review by artem-blazhko — APPROVED** [review]

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**